proto/regen/ecocredit/basket/v1/tx.proto

```diff
@@ -154,3 +158,20 @@ message MsgTakeResponse {
   // credits are the credits taken out of the basket.
   repeated BasketCredit credits = 1;
 }
+
+// MsgUpdateBasketFee is the Msg/UpdateBasketFee request type.
+message MsgUpdateBasketFee {
+
+  // root_address is the address of the signer.
+  // This MUST equal the address of the gov module for the tx to succeed.
+  string root_address = 1;
+
+  // add_fees defines a list of coins to append to the list of fees that can be used in the basket creation fee.
+  repeated cosmos.base.v1beta1.Coin add_fees = 2;
+
+  // remove_fee_denoms defines a list of denoms to remove from the list of basket creation fees.
+  repeated string remove_fee_denoms = 3;
+}
```

Comment:
As mentioned above, I think we want to support a single basket fee and this should update that single fee.